src/workflows-app/SubmissionDetails.js

```diff
@@ -400,6 +386,26 @@ export const BaseSubmissionDetails = ({ name, namespace, workspace, submissionId
                             return h(TextCell, [customFormatDuration(paginatedPreviousRuns[rowIndex].duration)]);
                           },
                         },
+                        {
+                          size: { basis: 400, grow: 0 },
+                          field: 'duration',
+                          headerRenderer: () => h(Sortable, { sort, field: 'duration', onSort: setSort }, ['Workflow ID']),
+                          cellRenderer: ({ rowIndex }) => {
+                            return h(
+                              Link,
+                              {
+                                href: Nav.getLink('workspace-workflows-app-run-details', {
+                                  namespace,
+                                  name,
+                                  submissionId,
+                                  workflowId: paginatedPreviousRuns[rowIndex].engine_id,
```

Comment:
Can we check that engine_id is defined before rendering the link? I remember us having a bug where if `engine_id` is undefined it breaks the whole page (this happens when CBAS has an issue submitting to Cromwell).

Reply:
Interesting, I thought that CBAS defined the engine id, and so it wasn't dependent on the Cromwell submission working, but I do vaguely remember what you're talking about. I'll check to make sure that's defined.

Reply:
I think CBAS only marks it in the database if we get a successful response from submitting to Cromwell (otherwise we have an engine ID that seemingly doesn't correspond to anything)